integrated_channels/moodle/client.py

```diff
@@ -409,12 +403,9 @@ def delete_content_metadata(self, serialized_data):
             rsp._content = bytearray('{"result": "Course not found."}', 'utf-8')  # pylint: disable=protected-access
             return rsp
         moodle_course_id = parsed_response['courses'][0]['id']
-        params = {
-            'wsfunction': 'core_course_update_courses',
-            'courses[0][id]': moodle_course_id,
-            'courses[0][visible]': 0  # hide a course rather than doing a true delete
-        }
-        response = self._wrapped_post(params)
+        serialized_data['wsfunction'] = 'core_course_update_courses'
+        serialized_data['courses[0][id]'] = moodle_course_id
```

Comment:
I would just confirm that moodle accepts all the other, not used fields being passed in through serialized data, otherwise we'll want to only return the values associated with the delete in `_apply_delete_transformation`

Reply:
You got it already on Slack 😉 🧠 

> oh wait it’s an update call not a delete call. so it makes sense to have all the other fields a part of the metadata with the call. ok yeah this should be fine but I’d still test it